### ISSUE TITLE
using quotes in default for table row gives multiple, nested quotes in create table

### ISSUE THREAD

**Issue by dalemcbeath** [sequelpro/sequelpro#3023]
V 1.1.2, build 4541 (602e11a)

entering a set of quotes (single or double) with blanks or no characters as row DEFAULT gives strange results in table info / create table...
```
CREATE TABLE `events` (
  `pk_id` int(100) NOT NULL AUTO_INCREMENT,
  `event_ID` varchar(100) NOT NULL DEFAULT '',
  `cal_ID` varchar(100) NOT NULL DEFAULT ''' ''',
  `CREATED` varchar(100) NOT NULL DEFAULT ''' ''',
  `UID` varchar(100) NOT NULL DEFAULT ''' ''',
  `DTEND` varchar(100) NOT NULL DEFAULT ''' ''',
  `SUMMARY` varchar(100) NOT NULL DEFAULT ''' ''',
  `DTSTART` varchar(100) NOT NULL DEFAULT ''' ''',
  `DTSTAMP` varchar(100) NOT NULL DEFAULT ''' ''',
  `SEQUENCE` varchar(100) NOT NULL DEFAULT ''' ''',
  `LOCATION` varchar(100) NOT NULL DEFAULT ''' ''',
  `DESCRIPTION` varchar(100) NOT NULL DEFAULT '" "',
  `ATTACH` varchar(11) DEFAULT '" "',
  `unknown` varchar(100) NOT NULL DEFAULT '" "',
  `error` tinyint(4) NOT NULL DEFAULT '0',
  PRIMARY KEY (`pk_id`),
  UNIQUE KEY `pk_id` (`pk_id`),
  UNIQUE KEY `event_ID` (`event_ID`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```
can't get rid of them.  select all on DEFAULT field and deleting still shows nested quotes in table info.

**Comment by dmoagx** [comment]
I don't understand your problem. If you enter a set of quotes with blanks as the default value, `''' '''` is exactly the expected result.

**Comment by dalemcbeath** [comment]
OK, if that's correct then it's just not what I'm used to from MySQLWorkbench.   

From workbench, if I set a row DEFAULT '', it stays ''.  (2 single quotes)
If i set DEFAULT to "",(2 double quotes) it changes it to 2 single quotes.
If I leave DEFAULT blank, it's an error.  

So having sequal pro automatically add 2 single quotes as default but not show them, seemed an error.
and getting back 6 single quotes when I entered only two seemed an error.  

Now that I know sequal pro actually adds quotes even if not shown if the field is "empty", then surrounds any input quotes with more quotes, it sort of makes sense.  

A way around this on an interface level might be to show the 2 single quotes greyed out (default text)  until the user enters a string, then replace that string with what is actually generated....the only way I found this strangeness was to look at table info.

Thanks for the clarification.
